### DISCUSSION TITLE
SSG on Vercel

### DISCUSSION THREAD

**Discussion by liconver** [nuxt/nuxt#17288] [Questions]
Is there a way to have Vercel run the generate command when building your site instead of the build command?  Is there anyway to customize the Vercel nitro preset?

**Comment by leonardssh** [comment]
Maybe https://nuxtjs.org/deployments/vercel/#static-site-with-vercel ?

Also, vercel allows you to customize the command it executes for building/generating. You can play around with a commands and presets and find the one that suits you.
![image](https://user-images.githubusercontent.com/35312043/178794387-2b2d1823-c612-41d2-b73c-24e016a95b50.png)
